django/cantusdb_project/main_app/management/commands/add_feasts.py

```diff
@@ -25,30 +26,34 @@ def handle(self, *args, **options):
             for element in feast:
                 element_name = element.tag
                 element_value = feast.find(element.tag).text
-                cleanr = re.compile("<.*?>")
-                element_value = re.sub(cleanr, "", element_value)
+                if element_value:
+                    cleanr = re.compile("<.*?>")
+                    element_value = re.sub(cleanr, "", element_value)
 
-                if element_name == "name":
-                    name = element_value
-                if element_name == "description":
-                    description = element_value
-                if element_name == "date":
-                    split_str = element_value.split(".")
-                    if len(split_str) > 1:
-                        day = int(split_str[1])
-                        month = int(datetime.strptime(split_str[0], "%b").month)
-                if element_name == "feast_code":
-                    if element_value != "xxx":
-                        feast_code = int(element_value)
-                if element_name == "id":
-                    feast_id = int(element_value)
+                    if element_name == "name":
+                        name = element_value
+                    if element_name == "description":
+                        description = element_value
+                    if element_name == "date":
+                        split_str = element_value.split(".")
+                        if len(split_str) > 1:
+                            day = int(split_str[1])
+                            month = int(datetime.strptime(split_str[0], "%b").month)
+                    if element_name == "feast_code":
+                        if element_value != "xxx":
+                            feast_code = int(element_value)
+                    if element_name == "id":
+                        feast_id = int(element_value)
+                    if element_name == "notes":
+                        notes = element_value
```

Comment:
Not sure how the interpreter handles this but if it doesn't do any optimization then `elif` might avoid unnecessary checks (unless you plan on modifying `element_name` in a branch here).

Reply:
Good point, I'll change it

Reply:
Changed it!